pyproject.toml

```diff
@@ -72,12 +73,24 @@ select = [
 ignore = [
     'E401',   # Multiple imports on one line
     'E402',   # Module level import not at top of file
-    'E501',   # Line too long
     'F541',   # f-string without placeholders
```

Comment:
We can remove F541 as well and fix all the places that violate this rule.

Reply:
Removed.  F541 was an intentional style choice.  I think there is nothing wrong with f-strings lacking placeholder.